sympy/solvers/ode/systems.py

```diff
@@ -351,7 +351,7 @@ def linear_ode_to_matrix(eqs, funcs, t, order):
     matrix differential equation [1]. For example the system $x' = x + y + 1$
     and $y' = x - y$ can be represented as
 
-    .. math:: A_1 X' = A0 X + b
+    .. math:: A_1 X' = A_0 X + b
```

Comment:
https://docs.sympy.org/latest/modules/solvers/ode.html#linear-ode-to-matrix

This indeed seems to be a typo (see the display in the docs).

Reply:
thx